docs/guide/yaml/test/example_yaml/paralleltestcase-example.yaml

```diff
@@ -0,0 +1,46 @@
+brooklyn.catalog:
+  id: parallel-test
+  version: 1.0
+  itemType: template
+  iconUrl: http://tomcat.apache.org/images/tomcat.png
+  name: Parallel Tomcat and DB Example Test
+  license: Apache-2.0
+  item:
+    brooklyn.config:
+      simple.confg: someValue
+    services:
+    - type: org.apache.brooklyn.test.framework.TestCase
+      name: Parallel Testcase
+      brooklyn.children:
+      - type: org.apache.brooklyn.test.framework.TestCase
+        name: Parallel Testcase
```

Comment:
Both having the name 'Parallel Testcase' is a mistake, the second should be something along the lines of 'Deploy Application'.

The first TestCase is used to allow sequential execution of the children, ie deploy app, then run tests. The ParallelTestCase starts the `ControlledDynamicWebAppCluster` and `MySqlNode` in parallel to reproduce the behaviour without tests (ie just nested under `-services:`.